File: src/main/java/systems/conduit/main/mixins/MinecraftServerThreadMixin.java
package systems.conduit.main.mixins;

import org.spongepowered.asm.mixin.Mixin;

@Mixin(targets = "net/minecraft/server/dedicated/DedicatedServer$2")
public class MinecraftServerThreadMixin implements Runnable {

    /**
     * Kills default server input thread in favor of ours.
     *
     * @author ConduitMC
     */
    @Override
    public void run() {
    }
}

File: src/main/kotlin/systems/conduit/core/mixins/MinecraftServerThreadMixin.kt
package systems.conduit.core.mixins

import org.spongepowered.asm.mixin.Mixin

@Mixin(targets = ["net/minecraft/server/dedicated/DedicatedServer$2"])
class MinecraftServerThreadMixin: Runnable {
    /**
     * Kills default server input thread in favor of ours.
     *
     * @author ConduitMC
     */
    override fun run() {}
}